spk/fuse-exfat/src/installer.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# Package
+PACKAGE="exfat"
+DNAME="exfat"
```

Comment:
Please use consistent names. `exFAT`, assuming that's what the project calls it.